specification/azsadmin/resource-manager/subscriptions/Microsoft.Subscriptions.Admin/preview/2015-11-01/Subscriptions.json

```diff
@@ -100,6 +100,80 @@
+                    },
+                    "Move user subscriptions from a Delegated Provider to the Default Provider.": {
+                        "$ref": "./examples/Subscriptions/MoveSubscriptionsToDefaultProvider.json"
+                    }
+                },
+                "description": "Move subscriptions between delegated provider offers.",
+                "tags": [
+                    "Subscriptions"
+                ],
+                "operationId": "MoveSubscriptions",
+                "parameters": [
+                    {
+                        "$ref": "#/parameters/SubscriptionIdParameter"
+                    },
+                    {
+                        "$ref": "#/parameters/ApiVersionParameter"
+                    },
+                    {
+                        "$ref": "#/parameters/MoveSubscriptionsParameter"
+                    }
+                ],
+                "responses": {
+                    "204": {
+                        "description": "NOCONTENT"
+                    },
+                    "202": {
+                        "description": "ACCEPTED"
+                    }
+                },
+                "x-ms-long-running-operation": true
+            }
+        },
+        "/subscriptions/{subscriptionId}/providers/Microsoft.Subscriptions.Admin/validateMoveSubscriptions": {
+            "post": {
+                "x-ms-examples": {
+                    "Validate that user subscriptions can be moved from the Default Provider to a Delegated Provider.": {
+                        "$ref": "./examples/Subscriptions/ValidateMoveSubscriptionsToDelegatedProvider.json"
+                    },
+                    "Validate that user subscriptions can be moved from a Delegated Provider to the Default Provider.": {
+                        "$ref": "./examples/Subscriptions/ValidateMoveSubscriptionsToDefaultProvider.json"
+                    }
+                },
+                "description": "Validate that user subscriptions can be moved between delegated provider offers.",
+                "tags": [
+                    "Subscriptions"
+                ],
+                "operationId": "ValidateMoveSubscriptions",
+                "parameters": [
+                    {
+                        "$ref": "#/parameters/SubscriptionIdParameter"
+                    },
+                    {
+                        "$ref": "#/parameters/ApiVersionParameter"
+                    },
+                    {
+                        "$ref": "#/parameters/MoveSubscriptionsParameter"
+                    }
+                ],
+                "responses": {
+                    "204": {
+                        "description": "NOCONTENT"
+                    },
+                    "202": {
+                        "description": "ACCEPTED"
+                    }
+                },
+                "x-ms-long-running-operation": true
```

Comment:
why there is no `200` response code described? Given operation is to validate a subscription can be moved, i would expect a result from service once the LRO is done. if user is expecting a result from the service then it needs to be modeled as schema of 200, as an example refer: https://github.com/Azure/azure-rest-api-specs/blob/cc5c0ef579d006a23a32685298e77015408d25ce/specification/compute/resource-manager/Microsoft.Compute/stable/2017-12-01/compute.json#L1412

Reply:
There is no 200 response by definition. This is consistent with Azure resource move operations.

Reply:
I can understand that's the case with operation `MoveSubscriptions`.

But name of this operation is `validateMoveSubscriptions` and descriptions says `Validate that user subscriptions can be moved between delegated provider offers`. If service is validating something how user will know the result of that validation?

Reply:
what i'm trying to understand is how service let user know that validation status? is there any error code service returns if move cannot be performed like 409 conflict.